four-cards.css

```diff
@@ -0,0 +1,90 @@
+* {
+    margin: 0;
+    font-family: sans-serif;
+    line-height: 1.5;
+}
+.header{
+    background-color: #F7F1FD;
+    padding: 80px 35px 35px 35px;
+    margin: auto;
+    text-align: center;
+}
+.lightH2 {
+    font-weight: lighter;
```

Comment:
We will learn in week 5 how to use Google Fonts. So, it would be better to use values (eg. 500, 800) as font-weight from then.